app/views/pages/about.html.erb

```diff
@@ -1 +1,5 @@
-<h1>About Us</h1>
+<h2>About Us</h2>
```

Comment:
@stephlaf 
Each page should have at least one `<h1>` tag for SEO!

Reply:
You're right. Let me change this!